app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt

```diff
@@ -333,6 +335,22 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_buttonState_isDisabled() {
+    launch<AdminAuthActivity>(
+      AdminAuthActivity.createAdminAuthActivityIntent(
+        context = context,
+        adminPin = "12345",
+        profileId = internalProfileId,
+        colorRgb = -10710042,
+        adminPinEnum = AdminAuthEnum.PROFILE_ADMIN_CONTROLS.value
+      )
+    ).use {
+      onView(withId(R.id.admin_auth_submit_button)).check(matches(not(isEnabled())))
```

Comment:
It would also be nice to test the opposite scenario where it is enabled correctly.

Reply:
Renamed to `testAdminAuthActivity_defaultButtonState_isDisabled `

And added one more test `testAdminAuthActivity_inputPin_buttonStateIsEnabled `